### ISSUE TITLE
Camel case support

### ISSUE THREAD

**Issue by SilverSheep** [thesayyn/protoc-gen-ts#78]
Now plugin outputs class getters and setters with the same name as in proto files.

It will be nice to have option to generate camelCase fields.

For instance for proto:

```
syntax = "proto3";

message Change {
    string long_name = 1;
}

```
have generated class:
```

export class Change {
			// omitted lines... 
   get longName() {
			// omitted lines... 
}
```

**Comment by thesayyn** [comment]
Hey.

Seems like this could be useful for the users.

I could point you in the right direction if you’d like to contribute.

**Comment by SilverSheep** [comment]
@thesayyn Why not? It was some time ago since I made the change in my project. I must recall what changes I commited and I'll let you know

**Comment by thesayyn** [comment]
fixed at #142 and released in [0.8.4](https://github.com/thesayyn/protoc-gen-ts/releases/tag/0.8.4)
